modules/cnf-understanding-workload-hints.adoc

```diff
@@ -9,7 +9,7 @@
 The following table describes how combinations of power consumption and real-time settings impact on latency.
 [NOTE]
 ====
-The following workload hints can be configured manually. See "Creating a performance profile" for further information. You can also work with workload hints using the Performance Profile Creator. See "Creating a performance profile" for further information.
+The following workload hints can be configured manually. You can also work with workload hints using the Performance Profile Creator. See the "Creating a performance profile" section for further information.
```

Comment:
ISG: https://www.ibm.com/docs/en/ibm-style?topic=references-citations "Use an italic font for the title unless the title is also a link, in which case use normal link highlighting."

```suggestion
The following workload hints can be configured manually. You can also work with workload hints using the Performance Profile Creator. See _Creating a performance profile_ for further information.
```

I would also suggest adding an `.Additional resources` section directly after in the assembly with a link to the section in question.

Reply:
@amolnar-rh I stand corrected, when referring to another section of the same document, quotes are correct.